.cirrus.yml

```diff
@@ -781,7 +737,12 @@ podman_machine_aarch64_task:
         PRIV_NAME: "rootless"  # intended use-case
         DISTRO_NV: "${FEDORA_AARCH64_NAME}"
         VM_IMAGE_NAME: "${FEDORA_AARCH64_AMI}"
-    clone_script: *get_gosrc_aarch64
+    # N/B: This script depends on ${DISTRO_NV} being defined for the task.
```

Comment:
This does not actually seem to be true

Reply:
This is most likely a leftover copy-and-paste "inclusion".  You're correct it can/should be removed.

Reply:
Opened https://github.com/containers/podman/pull/23112